arch/AVR/AVRDisassembler.c

```diff
@@ -0,0 +1,158 @@
+/* Capstone Disassembly Engine, http://www.capstone-engine.org */
+/* By Nguyen Anh Quynh <aquynh@gmail.com>, 2013-2022, */
+/* Rot127 <unisono@quyllur.org> 2022-2023 */
+/* VoltagedDebunked <rusindanilo@gmail.com> 2025 */
+
+#include <stdio.h>
+#include <string.h>
+#include <inttypes.h>
+
+#include "../../cs_priv.h"
+#include "../../MCInst.h"
+#include "../../MCInstrDesc.h"
+#include "../../MCFixedLenDisassembler.h"
+#include "../../MCRegisterInfo.h"
+#include "../../MCDisassembler.h"
+#include "../../utils.h"
+#include "AVRDisassembler.h"
+#include "AVRMapping.h"
+
+#ifdef CAPSTONE_HAS_AVR
+
+// Register decoder helpers
+static unsigned __attribute__((unused)) DecodeGPR8RegisterClass(MCInst *Inst, unsigned RegNo,
```

Comment:
```suggestion
static unsigned DecodeGPR8RegisterClass(MCInst *Inst, unsigned RegNo,
```

Also below. At lease before we merge it.